x-pack/plugins/security_solution/public/timelines/components/timeline/search_or_filter/translations.ts

```diff
@@ -84,9 +84,9 @@ export const RAW_EVENT = i18n.translate(
   }
 );
 
-export const ALERT_EVENT = i18n.translate(
-  'xpack.securitySolution.timeline.searchOrFilter.eventTypeAlertEvent',
+export const DETECTION_ALERTS_EVENT = i18n.translate(
+  'xpack.securitySolution.timeline.searchOrFilter.eventTypeDetectionAlertsEvent',
   {
-    defaultMessage: 'Alert events',
+    defaultMessage: 'Detection Alerts',
```

Comment:
Timeline changes looks good! Thanks for the extra `width`. 🙂  Curious if we'll want `alerts` lowercase to match events though? cc @lindseypoli @marrasherrier 

<p align="center">
  <img width="200" src="https://user-images.githubusercontent.com/2946766/86994015-9e98ef00-c162-11ea-8e36-815f85ee135d.png" />
</p>